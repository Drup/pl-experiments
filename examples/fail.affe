type channel : lin
val create : unit -> channel
val close : channel -> unit

let main =
  let c = create () in
  ()
