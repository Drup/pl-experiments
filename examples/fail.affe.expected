type channel = lin
create : unit -> channel
close : channel -> unit
Type error: The kind inequality lin < ^_k < affâ‚™ is not satisfiable.
